src/commands/lambda/constants.ts

```diff
@@ -1,34 +1,50 @@
 export const DD_LAMBDA_EXTENSION_LAYER_NAME = 'Datadog-Extension'
 export const EXTENSION_LAYER_KEY = 'extension'
+export const DOTNET_RUNTIME = 'dotnetcore3.1'
 export const RUNTIME_LAYER_LOOKUP = {
   [EXTENSION_LAYER_KEY]: DD_LAMBDA_EXTENSION_LAYER_NAME,
+  'dotnetcore3.1': 'poc-maxday-dotnet',
```

Comment:
We released the prod layer, the final name is `dd-trace-dotnet` so we can remove this placeholder name now